lib/internal/Magento/Framework/composer.json

```diff
@@ -45,7 +45,8 @@
             "Magento\\Framework\\": ""
         },
         "files": [
-            "registration.php"
+            "registration.php",
+            "Phrase/__function.php"
```

Comment:
I meant to just require `__.php` in registration of framework.

Reply:
To me the final goal should be to use Composer to supply autoloaded files. Using registration.php is an intermediate step which we could do without. It also keeps the purpose of the registration.

Reply:
> Using registration.php is an intermediate

There is no need to do something differently until we get rid of them.

So, instead of including `__.php` separately just do it within `registration.php`.